client/python/pyproject.toml

```diff
@@ -19,6 +19,22 @@ authors = [
 ]
 keywords = ["machine learning", "client", "API"]
 
+classifiers = [
+  'Development Status :: 5 - Production/Stable',
```

Comment:
Only nit here @WilliamRoyNelson is that we should probable use `Development Status :: 4 - Beta` as the client is still pre 1.0 version and some apis may change. Do you mind making the change?

Otherwise this looks good to me! Thanks again for the help.

Reply:
Thank you. I've made the requested change... and accidentally closed the PR while trying to rebase, but I think it's good and reopened now.

Reply:
Thanks @WilliamRoyNelson!